superset/advanced_data_type/plugins/eml_path.py

```diff
@@ -0,0 +1,78 @@
+import re
```

Comment:
I know there was a lot of discussion around this when Jan was doing his advanced type work and we went around in a few circles, but I think we want to keep the advanced types together in a deployment repo.

Reply:
Would you like for me to delete the eml_path.py file from [superset/advanced_data_type/plugins/]?

Reply:
In the midterm, we likely want to move our advanced types to Gitlab PB (since some types will contain PB information like stream names), but I believe this will take some additional tweaks to the deployment (e.g., loading the advanced types as a ConfigMap, perhaps, as per Tom's suggestion). This is a bigger change that's out of scope for your ticket. 

For now, I suggest we put the eml_path type in the hogwarts-superset-deployment repo (in [hogwarts-superset-deployment](https://github.com/CybercentreCanada/hogwarts-superset-deployment/tree/main)/[config](https://github.com/CybercentreCanada/hogwarts-superset-deployment/tree/main/config/advanced_data_types.py)) and also in https://gitlab.chimera.cyber.gc.ca/dasi2/dasi2c/superset-advanced-types . Jan has a branch on Gitlab which we should review and merge (but for now you can branch off epic/CLDN-2109). I'll raise a ticket to look into converting our Superset deployments to use the PB types. 

@cccs-joel , @cccs-RyanK -- does this plan sound reasonable?